priv/repo/migrations/20200710153620_create_delegations.exs

```diff
@@ -0,0 +1,29 @@
+defmodule LiquidVoting.Repo.Migrations.CreateDelegations do
+  use Ecto.Migration
+
+  def change do
+    create table(:delegations, primary_key: false) do
+      add :uuid, :uuid, primary_key: true
+
+      add :delegator_uuid,
+          references(:participants, column: :uuid, type: :uuid, on_delete: :delete_all)
+
+      add :delegate_uuid,
+          references(:participants, column: :uuid, type: :uuid, on_delete: :delete_all)
+
+      add :proposal_url, :text
+      add :organization_uuid, :uuid
+
+      timestamps()
+    end
+
+    create index(:delegations, [:delegator_uuid])
```

Comment:
same here and for the following two indices regarding making them composites with `:organization_uuid`